### ISSUE TITLE
Improve the OCF-wide configuration store

### ISSUE THREAD

**Issue by fydai** [ocf/projects#27]
https://ocf.io/d/52

**Comment by jvperrin** [comment]
I believe this is https://github.com/ocf/etc and it's mostly in service already

**Comment by cg505** [comment]
I guess there are more things to move into ocf/etc

**Comment by dkess** [comment]
I don't think we need an issue here, there are already issues filed in the etc repo for specific "improvements"

**Comment by fydai** [comment]
We would need to make changes in puppet and ocflib so I think this is deserving of a project issue.

**Comment by ethanhs** [comment]
Are there concrete things left to do for this?

**Comment by dkess** [comment]
All the issues on the etc repo are still valid.
